lib/makara/config_parser.rb

```diff
@@ -197,5 +197,15 @@ def recursive_sort(thing)
 
     end
 
+
+    def sanitize_id(id)
+      return if id.nil? || id.empty?
+
+      id.gsub(/[\|:]/, '').tap do |sanitized_id|
+        if sanitized_id.size != id.size
+          Logging::Logger.log "Proxy id '#{id}' changed to '#{sanitized_id}'", :warn
```

Comment:
Minor thing. Might want to say "Makara" here so they have more context.

Reply:
True. For some reason I had assumed the logger would tag the lines with `[Makara]`, but I realised it was added manually on exception handling. What I have done instead is to add this by default to all lines logged through `Makara::Logging::Logger.log`: https://github.com/taskrabbit/makara/pull/194/commits/51d32a580559893ba8d069e639772473780c9436  What do you think?